### DISCUSSION TITLE
Http server error in unity

### DISCUSSION THREAD

**Discussion by sislight224** [colyseus/colyseus#731] [General]
Hi, everybody. I have some errors to develop unity game.
I install this server local. so server was developed by Node and used http.
In unity side, if I create server, but unity can't join websocket...
and error message like this.

"Failed to make request to https://localhost:8080/matchmake/create/ufbRoom with secure protocol, trying again without!"
MatchMakeException: Unable to connect to the remote server

https://localhost:8080/matchmake/create/ufbRoom
This is problem. server is http. but this request is https..

**Comment by qhf566** [comment]
想找一款成熟的手游产品源码过来我们深度二次开发(全新换套美术，代码也都从新货混淆及优化整一遍)。意向最好是重度的mmo、arpg、角色扮演或动作手游，其他类型的游戏也可以，必须是高品质大厂近5年内做的产品，国内厂商或国外厂商开发的都可以，产品不能在国内运营过的。可以在国外运营过或运行中。准备了足够夸张的米给到你。如朋友那边有可以给推荐一下有米，不墨迹,鹅浩809996677。

**Comment by sislight224** [comment]
Hello, I am very interested in this role. actually, I have source code for
this, This is Boomerang Fu game source code. if you have any
questions, please let me know.
